app/src/main/java/com/mybus/service/MyBusService.java

```diff
@@ -0,0 +1,46 @@
+package com.mybus.service;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.mybus.builder.MyBusServiceUrlBuilder;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+public class MyBusService extends GenericService {
```

Comment:
For all the services we should extract the interface and interact with the service through it. This will make it easier to change the concrete implementation providing loose coupling.